libs/wingc/src/lib.rs

```diff
@@ -84,6 +84,36 @@ pub fn type_check(scope: &mut Scope, types: &mut Types) -> Diagnostics {
 		scope,
 		types,
 	);
+	add_builtin(
+		"assert",
```

Comment:
Lets avoid having these strings defined twice, something like:
```suggestion
		UtilityFunctionKind::Assert.to_string().as_str(),
```

Reply:
cac32eca312dcb0450370d25b930a10c6d859f9b